lib/rubocop/formatter/simple_text_formatter.rb

```diff
@@ -97,6 +87,55 @@ def message(offense)
         message = offense.corrected? ? green('[Corrected] ') : ''
         message << annotate_message(offense.message)
       end
+
+      # A helper class for building the report summary text.
+      class ReportSummarizer
+        include Colorizable, TextUtil
+
+        def initialize(file_count, offense_count, correction_count, context)
```

Comment:
Having an instances saves us from passing around variables on every method call. Also, I am not sure how well the `Colorizable` and `TextUtil` modules would work if I used class methods.

I’ll give it a shot if you insist :-)